src/js/config/Config.js

```diff
@@ -9,7 +9,7 @@ var Config = {
   networkingAPIPrefix: '/networking/api/v1',
   cosmosAPIPrefix: '/package',
   delayAfterErrorCount: 5,
-  documentationURI: 'https://dcos.io/docs',
+  documentationURI: 'https://dcos.io/docs/latest',
```

Comment:
Instead of `/latest` we should be hitting `/<dcos-version>`. There is currently not a clean way to get the DC/OS version in the correct format for the docs. The DC/OS version is at the `/dcos-metadata/dcos-version.json` endpoint but it returns version in `x.y.z` format rather than `x.y` needed for the docs site. 

I filed this ticket: https://dcosjira.atlassian.net/browse/DCOS-66